.bumpversion.cfg

```
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 1.0.0
+current_version = 1.0.1
```

Adding support for a new input argument would make this a feature, I think...


```suggestion
current_version = 1.1.0
```